src/core/components/pin.js

```diff
@@ -348,12 +348,12 @@ module.exports = (self) => {
               })
 
             if (!results.length) {
-              return cb(new Error(`Path is not pinned`))
+              return cb(new Error(`path '${paths}' is not pinned`))
```

Comment:
Changed to match error from go-ipfs
Interop test for this is added in https://github.com/ipfs/interface-js-ipfs-core/pull/375/files#diff-6a00eb1307f801bb37999815584589bdR190